Sources/PackageDescription/PackageDescription.swift

```diff
@@ -47,32 +48,29 @@ import Foundation
 ///         ]
 ///     )
 ///
-/// A `Package.swift` manifest file must begin with the string `//
-/// swift-tools-version:` followed by a version number specifier. The following code listing shows a few examples of valid declarations of the Swift tools version:
+/// The package manifest must begin with the string `// swift-tools-version:``,
+/// followed by a version number such as `// swift-tools-version:5.3.
 ///
-///     // swift-tools-version:3.0.2
-///     // swift-tools-version:3.1
-///     // swift-tools-version:4.0
-///     // swift-tools-version:5.0
-///     // swift-tools-version:5.1
+/// The Swift tools version declares:
 ///
-/// The Swift tools version declares the version of the `PackageDescription`
```

Comment:
Why are we dropping code voice for `PackageDescription`? It is a module name. I think we should spell it differently if we don't want to reference the module name here.

Reply:
We're not using code voice for names of frameworks as per style guide.